-1

Tested at: e479de140819a13cd11d6751543c0cb5bd243f88

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-24466/30260/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Plugins of all types refreshed.
gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_3_X_2018-09-04-1100'
gmake: *** [config/SCRAM/GMake/Makefile.rules:1692: src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>

--- a/RecoTracker/TkSeedGenerator/python/trackerClusterCheck_cfi.py
+++ b/RecoTracker/TkSeedGenerator/python/trackerClusterCheck_cfi.py
@@ -13,14 +13,14 @@ peripheralPbPb.toModify(trackerClusterCheck,
                         cut = "strip < 400000 && pixel < 40000 && (strip < 60000 + 7.0*pixel) && (pixel < 8000 + 0.14*strip)"
                         )
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
-    pp_on_XeXe_2017.toModify(trackerClusterCheck,
+pp_on_XeXe_2017.toModify(trackerClusterCheck,
                doClusterCheck=True, #FIXMETOO
                cut = "strip < 1000000 && pixel < 100000 && (strip < 50000 + 10*pixel) && (pixel < 5000 + strip/2.)",
                MaxNumberOfPixelClusters = 100000
                )
 
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-    pp_on_AA_2018.toModify(trackerClusterCheck,
+pp_on_AA_2018.toModify(trackerClusterCheck,
                doClusterCheck=True, 
                cut = "strip < 1000000 && pixel < 150000 && (strip < 50000 + 10*pixel) && (pixel < 5000 + strip/2.)",
                MaxNumberOfPixelClusters = 150000

